packages/shared/src/AdaptableBadge.js

```diff
@@ -1,4 +1,4 @@
-// @flow strict
+// @flow
```

Comment:
Why this change?

Reply:
It had it's reason, then I improved it, so yes, I will add it back tomorrow. Thanks 😊